### DISCUSSION TITLE
Stimulation coordinates

### DISCUSSION THREAD

**Discussion by FrancescoLomi** [simnibs/simnibs#396] [8. Simulation workflows] [answered]
Hi everybody,
What coordinates should I specify here? The world coordinates in mm of the point I stimulated or the corresponding MNI coordinates?
And why when I click on "project on scalp" the coordinates change?
![Schermata del 2024-04-10 15-40-40](https://github.com/user-attachments/assets/c4e8fcdd-fbd0-4eba-8597-282d6321d4b7)

Thank you so much

**Comment by bjornsig** [comment]
It is the center of the TMS coil or TES electrode on the skin surface in world space in mm. When you enter a position inside the head it will project the point to the surface.

**Comment by xiaoyu-TUD** [comment]
then we need to transform the MNI coordinate into the subject space?
simply using 'mni2subject_coords' command to transform coordinates from MNI space to subject space?

but if we still also need to make some changes of the orientation of the coil (TMS)? Specifically, the “y” axis is defined to be oriented according to the line from the center position to the reference position. should we also need to make some transformation of these 'y' axis values?

**Reply by TorgeW** [reply]
Yes that is all correct. 
It really much depends on what you want to achieve.
In general, you want to define the center position (possibly coming from MNI space) and the y axis position to orient the coil.

**Comment by FrancescoLomi** [comment]
Thank you!
I have a couple of other questions:
1) Why in this subject (see figure) the crosshair is in that position? It should be in in the left frontal cortex actually. I've inserted the world coordinates in mm of the brain surface and selected "project on scalp" but the software shows the point under the brain..
2) The rotation of the coil is indicated by the green line of the crosshair? So if the coil handle is orthogonal to the midline so should be the green line?
3) The spreading of the electric field in the "subject overlays" simulation is influenced by altered connectivity and/or atrophy or not? 
Thanks again!

![Schermata del 2024-04-10 18-33-25](https://github.com/user-attachments/assets/e37957ac-a663-4f76-a65e-127eeee527ad)

**Reply by oulap** [reply]
Hi Francesco,

1. Are you mapping these coordinates to the subject from the MNI space or are you defining those in the subject space directly? If you're mapping from MNI, please check that the coordinates are in the correct spot in the MNI template that is shipped with SimNIBS. If you're defining the coordinates in subject space, please make sure that the points are in the correct place in the T1.nii.gz file that's inside the m2m-folder.

2. As explained here:

https://simnibs.github.io/simnibs/build/html/tutorial/gui.html

The green axis shows the prolongation of the coil axis, so the coil handle basically points the other way. See also the coordinate system definition here:

https://simnibs.github.io/simnibs/build/html/_downloads/46a4ff127db96296fb4238660aec96f0/coil_axesorientation.pdf

Note the direction of the y-axis.

3. The simulations account for "geometrical" changes in the subject anatomy, but they do not account for micostructural changes of the tissue itself (unless you have diffusion MRI data). So the spatial distribution of the electric field is affected by atrophy, but not connectivity (again unless you have diffusion MRIs that can be used to locally change the electric conductivity).

**Comment by FrancescoLomi** [comment]
1) Yes i'm using the raw subject coordinates in mm, not MNI. I've inserted the coordinates that in the raw T1 corresponds to the brain region I stimulated but if i select "project on scalp" the crosshair goes elsewhere..
3) Is there a way to incorporate diffusion MRI data in the model? Because I have it!
Thanks!

**Reply by oulap** [reply]
Hi Francesco,

1. Could you post a screenshot from the GUI of where the coordinates land before you press project to scalp? If there is a large change from the initial position to the projected position, one reason might be that there's a skin surface closer to the target point somewhere else than directly above the cortex. This could be for example in the sinuses. If that's the case you would have to define the part of the scalp where the target point should be projected to.

2. Yes, please see this tutorial:

https://simnibs.github.io/simnibs/build/html/documentation/command_line/dwi2cond.html

**Reply by FrancescoLomi** [reply]
Thank you! Yes, without selecting project on scalp the crosshair is still in that strange position under the brain..
![Schermata del 2024-04-10 20-08-47](https://github.com/user-attachments/assets/89f28211-fcdd-49c1-bdb0-1f0eb1ce6fd1)

**Comment by oulap** [comment]
And you pick these coordinates using the T1.nii.gz file in the m2m-folder? How exactly did you pick the world coordinate? Was it in a viewer like freeview, or in python?

**Comment by FrancescoLomi** [comment, marked as answer]
I'm sorry, I found the error! I took the coordinates from another MRI sequence of the same subject. Now i've put the coordinated of the T1 sequence and it worked. Thank you so much!
